lib/mini_racer/truffleruby.rb

```diff
@@ -0,0 +1,305 @@
+# frozen_string_literal: true
+
+require "monitor"
+
+module MiniRacer
+
+  class Context
+    def heap_stats
+      {
+        total_physical_size: 0,
+        total_heap_size_executable: 0,
+        total_heap_size: 0,
+        used_heap_size: 0,
+        heap_size_limit: 0,
+      }
+    end
+
+    def stop
+      if @context.respond_to?(:stop)
+        if @entered
+          @context.stop
+          @stopped = true
+          stop_attached
+        end
+      end
+    end
+
+    private
+
+    @@context_initialized = false
+    @@use_strict = false
```

Comment:
cvars should be avoided, unless necessary for compatibility with the existing code?

Reply:
I've updated these to class instance variables.